app/models/concerns/with_assignments.rb

```diff
@@ -20,8 +20,8 @@ def has_messages_for?(user)
 
   # TODO: When the organization is used in this one, please change guide.pending_exercises
   # TODO: Please do the same on WithAssignmentsBatch
-  def find_assignment_for(user, _organization)
```

Comment:
It is time to fix those TO-DOs, isn't it?


I think that everything will break if we don't change `WithBatchAssignments` too.

Reply:
Noticed that during the integration test! fixed!